### DISCUSSION TITLE
consider not minifying packages

### DISCUSSION THREAD

**Discussion by mayank99** [storybookjs/storybook#23970] [Ideas]
### Describe the bug

all `@storybook` packages have their distributed code minified (presumably via `esbuild` + `tsup`). i understand the need for transpiling out typescript and maybe even bundling imports, but it would be nice to not have the code minified.

ultimately all code passes through a bundler/minifier (webpack, vite, etc) which will do its own minification, so i think there is no benefit to pre-minifying all packages.

### Additional context

we're always having all sorts of issues with storybook, and ultimately need to dig inside node_modules to find the issue. if the code was more readable, this would make our lives a lot easier.

**Comment by vanessayuenn** [comment]
@ndelangen @yannbf y'all did something recently related to this request, right?
